theoplayer/how-to-guides/01-ads/02-customizing-the-ad-overlay-text.md

```diff
@@ -85,20 +69,97 @@ If you are interested in changing the aspect of the overlay or its content, you
 }
 ```
 
-##### Legacy Android SDK (4.12.x)
+## Google IMA/DAI
+
+With Google IMA/DAI, it is not possible to directly edit the default UI. You can pass a locale to IMA to completely localize the ad UI. On the following sections we will explain the possibilities.
+
+### Changing or localizing the text
+
+It is not possible to edit the texts on the default UI on Google IMA/DAI SDKs. However, you can pass a locale to IMA to completely localize its ad UI like the following:
+
+##### Web SDK
+
+```javascript
+var element = document.querySelector('.video-js');
+const player = new THEOplayer.Player(element, {
+    license: "your_license_here",
+    libraryLocation: 'https://example.com',
+    ads: {
+        googleIma: {
+            language: "nl"
+        }
+    }
+});
+```
+You can find a list of locales supported by IMA SDK on web [here](https://developers.google.com/interactive-media-ads/docs/sdks/html5/client-side/localization#locale-codes).
+
+##### Android SDK
+
+You can pass [your preferred language](https://developers.google.com/interactive-media-ads/docs/sdks/android/client-side/api/reference/com/google/ads/interactivemedia/v3/api/ImaSdkSettings.html#setLanguage(java.lang.String)) while adding the Google IMA integration to the player:
+
+```kotlin
+private fun addGoogleImaIntegration() {
+    val settings = ImaSdkFactory.getInstance().createImaSdkSettings()
+    settings.language = "nl"
+
+    val imaIntegration = createGoogleImaIntegration(tpv!!, settings)
+
+    tpv?.player?.addIntegration(imaIntegration)
+}
+```
+You can find a list of locales supported by IMA SDK on Android [here](https://developers.google.com/interactive-media-ads/docs/sdks/android/client-side/localization#locale-codes).
+
+##### iOS SDK
+
+You can pass [your preferred language](https://developers.google.com/interactive-media-ads/docs/sdks/ios/client-side/reference/Classes/IMASettings#language) while adding the Google IMA integration to the player:
+
+```swift
+private func setupImaIntegration() {
+    let imaSettings = IMASettings()
+    imaSettings.language = "nl"
+        
+    let imaIntegration: GoogleImaIntegration = GoogleIMAIntegrationFactory.createIntegration(on: self.theoplayer!, with: imaSettings)
+
+    theoplayer.addIntegration(imaIntegration)
+    }
+```
+
+You can find a list of locales supported by IMA SDK on Android [here](https://developers.google.com/interactive-media-ads/docs/sdks/ios/client-side/localization#locale-codes).
+
+### Changing the styling
+
+##### Web SDK
+
+Currently it is not possible to customize the default UI on Google IMA/DAI SDKs on Web with the expection of toggling certain [UiElements](https://developers.google.com/interactive-media-ads/docs/sdks/html5/client-side/reference/js/google.ima#.UiElements). This can be done like the following:
```

Comment:
```suggestion
Currently it is not possible to customize the default UI on Google IMA/DAI SDKs on Web with the exception of toggling certain [UiElements](https://developers.google.com/interactive-media-ads/docs/sdks/html5/client-side/reference/js/google.ima#.UiElements). This can be done like the following:
```